resources/test/sse_data_schema.json

```diff
@@ -231,15 +231,31 @@
       ],
       "properties": {
         "hash": {
-          "$ref": "#/definitions/BlockHash"
+          "description": "`BlockHash`",
```

Comment:
Changes in this file seem unrelated to the PR. Did they end up here accidentally?

Reply:
This PR does add doc comments to several types. I made these changes to pass the sse schema test.